goqtdrv5/cdrv.h

```diff
@@ -60,8 +63,9 @@ typedef uint8_t goBool;
 
 
 #define drvInvalid (void*)-1
+#define GO_STRING_POINTER_SIZE 0x10
```

Comment:
This should be 8 bytes on i386 and arm (for reference, cgo provides types like GoString, intgo, ...).